src/engine/MaxConstraint.cpp

```diff
@@ -332,7 +376,7 @@ void MaxConstraint::resetMaxIndex()
 List<PiecewiseLinearConstraint::Fix> MaxConstraint::getPossibleFixes() const
 {
     ASSERT( !satisfied() );
-    ASSERT( _assignment.exists( _f ) && _assignment.size() > 1 );
+    ASSERT( _assignment.exists( _f ) && _assignment.size() > 1 ); // todo Guy A - 2nd assert is still relevant?
 
     double fValue = _assignment.get( _f );
     double maxVal = _assignment.get( _maxIndex );
```

Comment:
maxVal might now be the value of the eliminatedVariable, if it is greater than assignment[maxIndex]